src/Command/MarkMigrated.php

```diff
@@ -60,14 +61,33 @@ protected function configure()
             ));
         $this->addOption('plugin', 'p', InputArgument::OPTIONAL, 'The plugin the file should be created for')
             ->addOption('connection', 'c', InputArgument::OPTIONAL, 'The datasource connection to use')
-            ->addOption('source', 's', InputArgument::OPTIONAL, 'The folder where migrations are in');
+            ->addOption('source', 's', InputArgument::OPTIONAL, 'The folder where migrations are in')
+            ->addOption(
+                'target',
+                'u',
+                InputArgument::OPTIONAL,
+                'If present it will mark migrations from beginning to the given one'
+            )
+            ->addOption(
+                'exclude',
+                't',
```

Comment:
@dakota My bad, I forgot to update the short forms when I changed the options.

@HavokInspiration I will do the squashing and fix the short forms once we have a final way to implement the command/options.

I really liked the implementation having `--up-to` and `--exclude`. Having those two options we would fit into all possibilities without the need for deprecating anything. But anyway I'm up to you @dakota  and @HavokInspiration :)